test/fuzzing/run-shape-fuzzer-tests.py

```diff
@@ -68,11 +68,17 @@ def timeout(p, is_killed):
 fails = 0
 
 valgrind = None
+libtool = None
 if os.environ.get('RUN_VALGRIND', ''):
 	valgrind = which ('valgrind')
 	if valgrind is None:
 		print ("""Valgrind requested but not found.""")
 		sys.exit (1)
+	LIBTOOL = os.environ.get('LIBTOOL')
+	if LIBTOOL:
+		libtool = LIBTOOL.split(' ')[-1]
```

Comment:
Hack, as it was returning something '/bin/sh ../libtool' and I wanted the later part.

Reply:
Why?  Just leave it as is.

Reply:
I mean, split it and prepend that to the args list...